apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php

```diff
@@ -33,8 +33,10 @@ public function execute($request)
     $limit = sfConfig::get('app_hits_per_page', 10);
     $culture = $this->context->user->getCulture();
 
-    // Avoid pagination over 10000 records
-    if ((int)$limit * (int)$request->page > 1000)
+    // Avoid pagination over ES' max result window config (default: 10000)
+    $maxResultWindow = arElasticSearchPluginConfiguration::getMaxResultWindow();
+
+    if ((int)$limit * (int)$request->page > $maxResultWindow)
```

Comment:
I can't find any code style guides that specify whether there should be a space after a type casting operator, but most of the examples I've seen do include a space, e.g. https://www.php-fig.org/psr/psr-12/#61-unary-operators:

`$intValue = (int) $input;`

I prefer the space, but I will also accept the PR as-is, since there doesn't seem to be consensus either way.

Reply:
It looks like most of our code doesn't use the space, while the Symfony framework and Elastica lib from vendor do use the space. I'd rather let it as it is and see if the syntax checker does something about when we have it.